internal/provisioner/plain/README.md

```diff
@@ -134,24 +135,29 @@ resources referenced by the bundle are present on the cluster.
 
 ### Setup
 
-To experiment with the `plain` provisioner locally, first setup a local cluster, or simply
-have [kind](https://kind.sigs.k8s.io/) installed locally.
+To experiment with the `plain` provisioner locally, take the following steps to
+create a local kind cluster and deploy the provisioner onto it:
```

Comment:
It looks like we dropped the link to the kind documentation here. Is that still worth linking out to that documentation?

Reply:
I had removed it because we automatically use kind, but it might be nice to leave the link in so that people can read more about it if they're not familiar.